tests/test_bgp_af.rb

```diff
@@ -106,6 +110,41 @@ def test_collection_not_empty
     end
   end
 
+  def config_ios_xr_dependencies(asn, vrf='red', casty='unicast')
+    # These dependencies are required on ios xr
+
+    # router bgp 55
```

Comment:
Cleaned this up; loopback does not appear to be needed.

"casty" is just capturing the unicast/multicast of the address-family, it is only used in one place.  This will not be needed after migrating to a matrix-based test harness.